### PR TITLE
Refactor: refine the patient messages and add patient messages pages.

### PR DESCRIPTION
**Pull request by stu01509**
Fixes #

#### Short description of what this resolves:
Remove outdated code.
Adjust the bootstrap grid system and table style.
Adjust the style to follow the coding style.

image

image

image

### CODE REVIEW

**Comment by tywrenn** [comment]
@stu01509 Looks like you need to rebase your branch

**Comment by stu01509** [comment]
Thanks for your mention, resolved.

File: interface/patient_file/summary/pnotes_full.php [hunk 1]
```diff
@@ -276,469 +276,462 @@ function restoreSession() {
 }
 </script>
 </head>
-<body class="body_top">
+<body>
 
-<div class="container-fluid" id="pnotes"> <!-- large outer DIV -->
+<div class="container-fluid mt-3" id="pnotes"> <!-- large outer DIV -->
```
Comment (tywrenn) [inline interface/patient_file/summary/pnotes_full.php:281]: This feels like it should be a `container` instead of a `container-fluid`

Comment (stu01509) [reply interface/patient_file/summary/pnotes_full.php]: Updated the commit :)

File: interface/patient_file/summary/pnotes_full.php [hunk 2]
```diff
@@ -276,469 +276,462 @@ function restoreSession() {
...
+                                echo " <tr class='text billing'>\n";
+                                echo "  <td>" . $colorbeg . xlt('Balance Due') .
+                                    $colorend . "&nbsp;" . $colorbeg . text($formatted) .
+                                    $colorend . "</td>\n";
+                                echo " </tr>\n";
+                            }
+
+                            if ($billing_note) {
+                                echo " <tr class='text billing'>\n";
+                                echo "  <td>" . $colorbeg . xlt('Billing Note') .
```
Comment (stu01509) [inline interface/patient_file/summary/pnotes_full.php:361]: Hi @tywrenn 

I am not sure, but I guess the `.text` the class has been deprecated, Should I remove it?

Comment (tywrenn) [reply interface/patient_file/summary/pnotes_full.php:361]: That actually does a font resize, found in `/interface/themes/core/text.scss`. It isn't deprecated *yet*.

**Comment by stu01509** [comment]
Updated the commit.

image

**Comment by tywrenn** [comment]
@stu01509 Travis isn't happy

**Comment by stu01509** [comment]
Update the commit. Travis is happy now.

File: interface/patient_file/summary/pnotes_full.php [hunk 3]
```diff
@@ -276,467 +276,461 @@ function restoreSession() {
...
-                $colorbeg = "<span style='color:red'>";
-                $colorend = "</span>";
-            }
+                //Display what the patient owes
+                $balance = get_patient_balance($patient_id);
+                ?>
+
+                <?php if ($billing_note || $balance) { ?>
+                    <div class="mt-1">
+                        <table class="table">
```
Comment (tywrenn) [inline interface/patient_file/summary/pnotes_full.php:346]: `table-responsive` here

Comment (stu01509) [reply interface/patient_file/summary/pnotes_full.php:346]: Update the commit.

**Comment by bradymiller** [comment]
hi @stu01509 , bringing in. thanks for the improvements! -brady

**Pull request merged**